### PR TITLE
chore: Update contributing and license links to correct location

### PR DESCRIPTION
**Pull request by JoanaParaiba**
# PR Details

Update contributing and license links to the correct location

## Description

Contributing guidelines link changed to '(./docs/CONTRIBUTING.md)'
License link changed to '(./LICENSE)' 

## Related Issue

https://github.com/Farfetch/react-context-responsive/issues/15#issue-672896953

## How Has This Been Tested

Open the repository, and go to the Contributing section. The link should work.

## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING](../docs/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### Disclaimer

By sending us your contributions, you are agreeing that your contribution is made subject to the terms of our [Contributor Ownership Statement](https://github.com/Farfetch/.github/blob/master/COS.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -168,7 +168,7 @@ The non-hook APIs just expose the `useResponsive` hook with different APIs, for
 
 ## Contributing
 
-Read the [Contributing guidelines](../docs/CONTRIBUTING.md)
+Read the [Contributing guidelines](https://github.com/Farfetch/react-context-responsive/blob/master/docs/CONTRIBUTING.md)
```
Comment (SoaresMG) [inline README.md:171]: It shouldn't be an absolute path

Comment (JoanaParaiba) [reply README.md]: You're right 👍

I've changed the [Contributing guidelines] link path to (./docs/CONTRIBUTING.md) and I've also changed [MIT] link path to (./LICENSE)

(Sorry for all the commits 😅  )

Comment (joaopedrodcf) [reply README.md]: Maybe you could squash every commit

**Comment by SoaresMG** [comment]
Just saw your commit message, it doesn't comply with commitlint settings, have you used `--no-verify` when creating the commit?

**Comment by JoanaParaiba** [comment]
No, I didn't use `--no-verify` when creating the commit

**Pull request merged**